docs/git-commands.md

```diff
@@ -0,0 +1,94 @@
+
+```
+git checkout -b <<desciptive-name-here>>
+```
+Now you are working on a new branch, when you are happy with your changes or just want to save the changes to the branch you can commit it, it is good practice to run the following command first to see what files have been changed by your work.
+```
+git status
+```
+If all the files listed are the ones you are happy with to commit to your branch run the following series of commands.
+```
+git add .
+git commit -m '<<describe the changes here>>'
+git push <<your-remote-name>> <<your-branch-name>>
+```
+Now you have pushed your changes on your branch to your remote, refer to tutorial.md on how to create a PR from here
+
+## Checking out other peoples PR's
+In order to check out someones PR you need to add that persons remote, this can be done with the following command, just replace the tags with the persons github name
+```
+git remote add <<persons-github-name>> https://github.com/<<persons-github-name>>/SchoolWebsite.git
+```
+Now that the person who created the PR's remote is added to your git project, you will need to fetch that person's branchs, that can be done with the following command after you have added their remote
+```
+git fetch <<persons-github-name>>
+```
+Now you have access to the branchs associated with that person's fork of the repo, if you run the following
+```
+git branch -v
+```
+You will see a list of available branchs to you locally, you will now need to check the PR on github to see what branch the PR was created on, once you got the name you can enter the following 
+```
+git checkout <<PR-branc-name>>
+```
+Now you can view and run the code of the pr locally. Once you are done you can checkout a different branch or if you wish to return to your own branch you where on before you checked out the PR run the following command
+```
+git checkout -
+```
+## Rebasing your own PR
+So with multiple people working on a project the master branch (the original repo) can get ahead of the branch you are working on, so you will need to catch your work up to the master, this is known as Rebasing. 
+
+This process is done by the following, first you need to checkout master, master is branch that is inline with the origin repo on github, once you are on the master branch you will need to pull the changes down from github. Now your master branch locally is uptodate with the github repo. Next you need to checkout the branch you want to rebase, once on this branch you can rebase off your local master branch. So to do that you can run the following commands
+```
+git checkout master
+git pull origin
+git checkout -
```

Comment:
It might be clearer to add `<<branch-to-rebase>>` here in case the reader is not on the branch they want to rebase before checking out master. Edge case but possible